package.json

```diff
@@ -41,7 +41,6 @@
     "@babel/core": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
     "babel-jest": "^24.7.1",
-    "insert-text-at-cursor": "^0.2.0",
```

Comment:
It'd be good to see if we could link to a github fork of this rather than include this dependency manually (if possible)

Reply:
Forked the original repo and updated with the code as it is in this PR:
https://github.com/alphagov/insert-text-at-cursor/pull/1 After I get it approved I will update this to use the code from a branch rather then vendoring it.

Reply:
The PR got merged in the original repo and a 3.0 release will include the fix. Will update to consume that.